nxdrive/gui/application.py

```diff
@@ -1529,7 +1570,15 @@ def update_status(self, engine: Engine) -> None:
 
     @pyqtSlot()
     def refresh_transfers(self) -> None:
-        transfers = self.api.get_transfers()
```

Comment:
Nope, nope, nope :)

Add a specific method for Direct Transfer items. Do not interfere here.

Reply:
Finally, you have to use it but the code logic must be adapter. As of now, this is not acceptable :)

But you can make use of the new `upload.is_direct_transfer` attribute now.